http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/NonLeoMessageSubscriber.scala

```diff
@@ -5,48 +5,26 @@ import cats.effect.Async
 import cats.effect.std.Queue
 import cats.mtl.Ask
 import cats.syntax.all._
-import org.broadinstitute.dsde.workbench.google2.DeviceName
-import org.broadinstitute.dsde.workbench.leonardo.db.{
-  appQuery,
-  clusterQuery,
-  kubernetesClusterQuery,
-  persistentDiskQuery,
-  DbReference,
-  RuntimeConfigQueries
-}
-import com.google.protobuf.ByteString
-import com.google.pubsub.v1.PubsubMessage
 import fs2.Stream
-import io.circe.syntax._
 import io.circe.{Decoder, DecodingFailure, Encoder}
 import org.broadinstitute.dsde.workbench.google2.JsonCodec.traceIdDecoder
-import org.broadinstitute.dsde.workbench.google2.{
-  Event,
-  GoogleComputeService,
-  GooglePublisher,
-  GoogleSubscriber,
-  ZoneName
-}
+import org.broadinstitute.dsde.workbench.google2.{DeviceName, GoogleComputeService, ZoneName}
 import org.broadinstitute.dsde.workbench.leonardo.AsyncTaskProcessor.Task
 import org.broadinstitute.dsde.workbench.leonardo.ErrorAction.DeleteNodepool
 import org.broadinstitute.dsde.workbench.leonardo.JsonCodec._
 import org.broadinstitute.dsde.workbench.leonardo.dao.{SamDAO, UserSubjectId}
-import org.broadinstitute.dsde.workbench.leonardo.http.dbioToIO
-import org.broadinstitute.dsde.workbench.leonardo.monitor.NonLeoMessage.{
-  CryptoMining,
-  CryptoMiningScc,
-  DeleteKubernetesClusterMessage,
-  DeleteNodepoolMessage
-}
+import org.broadinstitute.dsde.workbench.leonardo.db._
+import org.broadinstitute.dsde.workbench.leonardo.http.{ctxConversion, dbioToIO}
+import org.broadinstitute.dsde.workbench.leonardo.model.LeoAuthProvider
+import org.broadinstitute.dsde.workbench.leonardo.monitor.NonLeoMessage.{CryptoMining, CryptoMiningScc, DeleteKubernetesClusterMessage, DeleteNodepoolMessage}
 import org.broadinstitute.dsde.workbench.leonardo.monitor.NonLeoMessageSubscriber.cryptominingUserMessageEncoder
 import org.broadinstitute.dsde.workbench.leonardo.util.{DeleteClusterParams, DeleteNodepoolParams, GKEAlgebra}
 import org.broadinstitute.dsde.workbench.model.TraceId
 import org.broadinstitute.dsde.workbench.model.google.GoogleProject
 import org.broadinstitute.dsde.workbench.openTelemetry.OpenTelemetryMetrics
-import org.typelevel.log4cats.StructuredLogger
-import org.broadinstitute.dsde.workbench.leonardo.http.ctxConversion
-import org.broadinstitute.dsde.workbench.leonardo.model.LeoAuthProvider
 import org.broadinstitute.dsde.workbench.util2.InstanceName
+import org.broadinstitute.dsde.workbench.util2.messaging.{CloudPublisher, CloudSubscriber, ReceivedMessage}
+import org.typelevel.log4cats.StructuredLogger
```

Comment:
there are a lot of import formatting with this PR - are your settings the same as everyone else's? or are these files unformatted for some time? 

... I am asking because explicit imports are being re-written with `_` and list styles are different (just trying to avoid thrashing in format styles)